### ISSUE TITLE
force as compilation errors when students use variable length array (dynamic size at runtime)

### ISSUE THREAD

**Issue by remisharrock** [France-ioi/taskgrader#95]
is it possible to add an option to  force as compilation errors when students use variable length array (dynamic size at runtime)

for example:

https://github.com/remisharrock/c-programming-with-linux-MOOC-issues-tracker/issues/674

maybe it is something like this or gcc, but do we use gcc or clang ?

-Wpedentic -Werror

**Comment by mblockelet** [comment]
Hello, we do use gcc. Compared to the other issue, this one is trickier, as I don't think we can enable `-Wpedantic` globally ; it'd need to be an option you would have to set up on every task. Would you be okay with that?

Also, to answer the question in the linked bugreport : yes, with taskgrader it's possible to test a function, you just need to pass the solution as a dependency to a main C file you have.

**Comment by remisharrock** [comment]
Yes could you add the option and we will check it wherever we want. 
Is that ok for you ?

**Comment by mblockelet** [comment]
Sure, I will add it soon, probably this week ; I was unavailable in the past days and still pretty busy.
